src/Microsoft.DotNet.Build.Tasks.Feed.Tests/PublishArtifactsInManifestTests.cs

```diff
@@ -19,24 +19,25 @@ public class PublishArtifactsInManifestTests
         const string BlobFeedUrl = "https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json";
 
         [Fact]
-        public void FeedConfigParserTests1()
+        public async Task FeedConfigParserTests1()
         {
             var buildEngine = new MockBuildEngine();
             var task = new PublishArtifactsInManifest
             {
-                // Create a single ITaskItem for a simple feed config, then parse to FeedConfigs and
+                // Create a single IMicrosoft.Build.Utilities.TaskItem for a simple feed config, then parse to FeedConfigs and
```

Comment:
```suggestion
                // Create a single Microsoft.Build.Utilities.TaskItem for a simple feed config, then parse to FeedConfigs and
```

OR simply "Create a single TaskItem" don't think the whole type is all that interesting in the comment.

Reply:
Yeah, because System.Threading.Tasks.Task and Microsoft.Build.Utilities.Task overlap.